src/content/docs/dropins/blueprint/add-enrichment.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Enrich the DROPIN_NAME dropin
```

Comment:
@fnhipster wasn't enrichment just done through Slots? We can probably move this to a subsection within that blueprint if that is the case.

Reply:
It depends. I've seen examples of "content enrichment" where extra content is added below the dropin (and not necessary in a slot). That said, I do think this is more on the general "Extend  a Dropin" or "Enrich a Dropin" not so much on the individual case of each dropin.

Reply:
Yeah, that would be nice. I've always had a problem with the whole "enrichment" feature. It's just too vague. :)

Reply:
Cool. I'll remove this topic and add to the generalized Extending Dropins topic. thanks!